test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMultipartMixedBatchReaderTests.cs

```diff
@@ -205,6 +206,42 @@ public void ReadMultipartMixedBatchRequest()
             Assert.Empty(verifyResourceStack);
         }
 
+        [Fact]
+        public void ReadMultipartMixedBatchRequestWthoutContentId()
+        {
+            var payload = Regex.Replace(batchRequestPayload, "Content-ID: .", "");
+            Assert.NotEqual(payload, batchRequestPayload);
+            SetupMultipartMixedBatchReaderAndRunTest(
+                batchRequestPayload,

```

Comment:
Here you're passing the `batchRequestPayload` which contains the `Content-ID` header. Shouldn't you pass the `payload` variable instead, which does not have the `Content-ID` header?